Navigation/Profile/PostTableViewCell.swift

```diff
@@ -0,0 +1,161 @@
+    
+    private let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14, weight: .regular)
+        label.textColor = .systemGray
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private let likesLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+        label.textColor = .black
+        return label
+    }()
+    
+    private let viewsLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+        label.textColor = .black
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        setUpPostTableViewCell()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        
+        setUpPostTableViewCell()
+    }
+    
+}
+
+// MARK: Layout
+private extension PostTableViewCell {
+    func setUpPostTableViewCell() {
+        [authorLabel,
+         postImageView,
+         descriptionLabel,
+         likesLabel,
+         viewsLabel].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            
+            contentView.addSubview($0)
+        }
+        
+        NSLayoutConstraint.activate([
+            authorLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 10),
+            authorLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            authorLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            
+            postImageView.topAnchor.constraint(equalTo: authorLabel.bottomAnchor, constant: 10),
+            postImageView.widthAnchor.constraint(equalTo: contentView.widthAnchor),
+            postImageView.heightAnchor.constraint(equalTo: contentView.widthAnchor),
+            
+            descriptionLabel.topAnchor.constraint(equalTo: postImageView.bottomAnchor, constant: 10),
+            descriptionLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            descriptionLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            
+            likesLabel.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: 10),
+            likesLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            likesLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16),
+            
+            viewsLabel.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: 10),
+            viewsLabel.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            viewsLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16)
+        ])
+    }
+}
+
```

Comment:
Remove all unneeded code and commented code below.
File should end with one empty line after last }

Reply:
Done.